cassandra/src/test/java/org/hibernate/ogm/datastore/cassandra/utils/CassandraTestHelper.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.utils;
+
+
+import org.hibernate.SessionFactory;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.ogm.cfg.OgmConfiguration;
+import org.hibernate.ogm.datastore.cassandra.CassandraDialect;
+import org.hibernate.ogm.datastore.cassandra.impl.CassandraDatastoreProvider;
+import org.hibernate.ogm.datastore.document.options.AssociationStorageType;
+import org.hibernate.ogm.datastore.spi.DatastoreProvider;
+import org.hibernate.ogm.dialect.spi.GridDialect;
+import org.hibernate.ogm.model.key.spi.EntityKey;
+import org.hibernate.ogm.options.navigation.GlobalContext;
+import org.hibernate.ogm.utils.TestableGridDialect;
+
+import org.hibernate.persister.collection.CollectionPersister;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.ColumnDefinitions;
+import com.datastax.driver.core.DataType;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ProtocolVersion;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import com.datastax.driver.core.querybuilder.Select;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.eq;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.quote;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.select;
+
+/**
+ * Utility methods for test suite support.
+ *
+ * @author Jonathan Halliday
+ */
+public class CassandraTestHelper implements TestableGridDialect {
```

Comment:
Could you add an element for Cassandra to `GridDialectType`? That construct is not perfect, but it allows us to skip specific tests on certain backends.